core/unit_test/TestAbort.hpp

```diff
@@ -103,5 +103,14 @@ void test_abort_from_device() {
 
 TEST(TEST_CATEGORY_DEATH, abort_from_device) {
   ::testing::FLAGS_gtest_death_test_style = "threadsafe";
+// FIXME_OPENACC FIXME_NVHPC: NVHPC fails when targetting CPUs.
+#if defined(KOKKOS_ENABLE_OPENACC) && defined(KOKKOS_COMPILER_NVHPC) && \
+    defined(KOKKOS_ENABLE_OPENACC_FORCE_HOST_AS_DEVICE)
+  if (std::is_same<ExecutionSpace, Kokkos::Experimental::OpenACC>::value) {
```

Comment:
Will conflict with #7227
Maybe use is_same_v

Reply:
Updated to `is_same_v`

Reply:
I meant just this occurrence that was added, not the other two that were already there

Reply:
Undo the changes for the other two already-existing ones.